pkg/handlers/generic/mutation/imageregistries/credentials/inject_test.go

```diff
@@ -392,22 +394,44 @@ var _ = Describe("Generate Image registry patches", func() {
 
 	// Create credentials secret before each test
 	BeforeEach(func(ctx SpecContext) {
-		client, err := helpers.TestEnv.GetK8sClient()
+		client, err := helpers.TestEnv.GetK8sClientWithScheme(clientScheme)
 		gomega.Expect(err).To(gomega.BeNil())
+
 		gomega.Expect(client.Create(
 			ctx,
 			newRegistryCredentialsSecret(validSecretName, request.Namespace),
 		)).To(gomega.BeNil())
+
+		gomega.Expect(client.Create(
```

Comment:
I like explicitly creating cluster in unit test. I could use this pattern in encryption handler unit tests to verify secret ownership.

Reply:
wdyt moving the cluster create and delete to common test utils package?

Reply:
Yes, I want to do that. I tried it, then decided to do it as a follow-up PR.

Reply:
sounds good. thanks for making this improvement.